pkg/console/operator/sync_v400.go

```diff
@@ -71,12 +71,28 @@ func (co *consoleOperator) sync_v400(ctx context.Context, controllerContext fact
 		return statusHandler.FlushAndReturn(routeErr)
 	}
 
+	authnConfig, err := co.authnConfigLister.Get("cluster")
+	if err != nil {
+		return statusHandler.FlushAndReturn(err)
+	}
+
+	var authServerCAConfig *corev1.ConfigMap
+	if len(authnConfig.Spec.OIDCProviders) == 1 {
```

Comment:
`s/== 1/> 0/` would keep this working in the old mode when we make it possible to configure multiple providers